### PR TITLE
docs(release-highlights): Adds additional updates for v6 release.

### LINKED ISSUE #4289: Update PatternFly 6 release highlights
Update highlights as needed for the final release. Integrate anything missing, for example chatbot details

### PR DESCRIPTION
**Pull request by edonehoo**
Closes #4289 

Will need input on anything to add / remove, please and ty!

### CODE REVIEW

**Comment by tlabaj** [comment]
Did we add a note about prefixing the react-tokens with `t_` anywhere?  That is new for final release.

**Comment by edonehoo** [comment]
@tlabaj just added! Do you think it's clear to mention that following this heading structure, or too muddy?

> ### Tokens updates
> #### Design tokens
> #### React tokens : here

Is there any specific reasoning we should mention? I looked through some PRs and Slack threads that were related, but couldn't get a great sense of the reason it needed to happen

**Comment by andrew-ronaldson** [comment]
We'll have to rename the Battery component to Severity. We are using our new severity icons now

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 1]
```diff
@@ -7,6 +7,89 @@ section: get-started
...
+| Error communication | - Error boundary <br /> - Error state <br /> - Missing page (previously named "invalid object") <br /> - Unauthorized access (previously named "not authorized") <br /> - Unavailable content <br /> - Warning modal |
+| Helpers | - Column management modal <br /> - Log snippet <br /> - Shortcut grid |
+| Status and state indicators | - Ansible <br /> - Severity (previously Battery) <br /> - Skeleton table <br /> - Status (new!) <br /> - Tag count |
+
+### React component updates 
+
+Here are the most significant updates we made to our React components: 
+
+- Data list
+    - Refactored `<DataListCheck>` to use `<Checkbox>` internally. As a result, `id` is now a required prop for ` <DataListCheck>`. This also fixes broken checkboxes in the examples.
```
Comment (tlabaj) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:45]: @kmcfaul, didn't we update this so id is not required?

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 2]
```diff
@@ -7,6 +7,89 @@ section: get-started
...
+| Helpers | - Column management modal <br /> - Log snippet <br /> - Shortcut grid |
+| Status and state indicators | - Ansible <br /> - Severity (previously Battery) <br /> - Skeleton table <br /> - Status (new!) <br /> - Tag count |
+
+### React component updates 
+
+Here are the most significant updates we made to our React components: 
+
+- Data list
+    - Refactored `<DataListCheck>` to use `<Checkbox>` internally. As a result, `id` is now a required prop for ` <DataListCheck>`. This also fixes broken checkboxes in the examples. 
+- Dropdown
```
Comment (tlabaj) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:46]: So we kind of reverted this.  We have a prop to set the time out but it is `0` by default.  We added some other logic internally to stop the scroll from jumping when focusing the first item in the menu.

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 3]
```diff
@@ -7,6 +7,89 @@ section: get-started
...
+- [About PatternFly AI](/patternfly-ai/about-patternfly-ai)
+- [AI guidelines](/patternfly-ai/ai-guidelines)
+- [Chatbot](/patternfly-ai/chatbot/about-chatbot)
+- [Conversation design](/patternfly-ai/conversation-design)
+
+As with all things AI currently, this area will continue to grow and mature with time.
+
+### Component groups restructuring
+
+We made many updates to our component groups extension, to improve accuracy, usability, and alignment with PatternFly 6. We've moved its website section to our top-level navigation for better visibility, and also to support necessary sub-navigation. We've conceptually grouped the components into functional categories. Additionally, we renamed some of the components to be more accurate.
```
Comment (srambach) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:28]: Nitpicky nit - I don't think there should be a comma at `extension, to improve`

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 4]
```diff
@@ -7,6 +7,89 @@ section: get-started
...
+    - Fixed improper offset in [demo](/components/jump-links/react-demos). 
+- Menu toggle
+    - Removed `pf-m-actions` and, consequently, `SplitButtonOptions`. Items should now be passed directly to `splitButtonItems`.
+    - Added `isPlaceholder` to support customizable placeholder text, as well as [a corresponding example](/components/menu-toggle#placeholder-text-in-toggle).
+
+### Token updates 
+
+#### Design tokens 
+
+We updated existing design tokens and added new tokens to support directional box-shadows. To see these tokens, search "box-tokens" in our [tokens documentation](/tokens/all-patternfly-tokens).
```
Comment (srambach) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:58]: I think you'd want to search `"box-shadow"` not `-tokens` correct?

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 5]
```diff
@@ -7,6 +7,89 @@ section: get-started
...
+
+We also added a few new tokens to support the needs of our new chatbot, including:
+- A tertiary background token that accommodates containers placed on a secondary background.
+- Inverse hover and inverse clicked tokens.
+
+#### React tokens
+
+To address instances where chart tokens and chart variable names were unintentionally identical, we added a `t_` prefix to our React tokens. This makes it easier to differentiate between tokens and variables.
```
Comment (srambach) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:66]: Maybe say `between design tokens and other CSS variables.` or whatever that actually is, but it's in a React tokens section and referring to design token variables.

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md (hunk 1 above)
Comment (tlabaj) [reply packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:45]: I just verified.  The id is not required.

**Review by tlabaj — COMMENTED** [review]
can we add a bullet about [feat(TextInputGroup): added validation support](https://github.com/patternfly/patternfly-react/pull/10815)

**Review by tlabaj — CHANGES_REQUESTED** [review]
We should add information about deprecating `tile` and using `Card` instead.
https://github.com/patternfly/patternfly-react/pull/10821

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md (hunk 2 above)
Comment (edonehoo) [reply packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:46]: ahh okay, is that `focusTimeoutDelay`? How's this

> - Dropdown
>     - Added `focusTimeoutDelay`, which specifies the time to wait before setting focus on the first dropdown item when `shouldFocusFirstItemOnOpen` is set. 
>     - Fixed issues with invalid and jumpy scrolling when focusing on the first menu item.

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md (hunk 5 above)
Comment (mcoker) [reply packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:66]: You got it! That update lgtm.

> "we added a t_ prefix to all design tokens in React tokens"
> the prefix wasn't added to the React tokens themselves? I thought this part was the React token, am I misunderstanding?

Yep it's on the react token. When I sad "all design tokens in react tokens" I'm thinking of it like react tokens are a thing themselves, and are made up of design tokens and CSS vars. So with _design token in react tokens_, I'm just referring to any of the `t_` react tokens.

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 6]
```diff
@@ -7,6 +7,89 @@ section: get-started
...
+- [Chatbot](/patternfly-ai/chatbot/about-chatbot)
+- [Conversation design](/patternfly-ai/conversation-design)
+
+As with all things AI currently, this area will continue to grow and mature with time.
+
+### Component groups restructuring
+
+We made many updates to our component groups extension, to improve accuracy, usability, and alignment with PatternFly 6. We've moved its website section to our top-level navigation for better visibility, and also to support necessary sub-navigation. We've conceptually grouped the components into functional categories. Additionally, we renamed some of the components to be more accurate. 
+
+[Check out the updated documentation](/component-groups/about-component-groups), which includes:
```
[the remark this answers is not in the document]
Comment (edonehoo) [reply packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:30]: Eh, not a "super nit", this is important! And something I try to keep in mind, but definitely something I miss often when I'm rushing/spreading myself thin. Even then I tend to think of it from a "it should be obvious what this link is" vs the screenreader perspective, so this is a good reminder about that other side. I'll update this one to your suggestion and also do a quick look over 👀

**Comment by edonehoo** [comment]
>  we also added `isPlaceholder` prop to the `EditableSelectInputCell` of table [patternfly/patternfly-react#10850](https://github.com/patternfly/patternfly-react/pull/10850)

@tlabaj  is this only for deprecated table?

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md (hunk 2 above)
Comment (tlabaj) [reply packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:46]: Sounds good @edonehoo

**Comment by tlabaj** [comment]
> > we also added `isPlaceholder` prop to the `EditableSelectInputCell` of table [patternfly/patternfly-react#10850](https://github.com/patternfly/patternfly-react/pull/10850)
> 
> @tlabaj is this only for deprecated table?

you are right, it is for deprecated table. So we don't have to mention that.

**Review by tlabaj — APPROVED** [review]

**Comment by nicolethoen** [comment]
@edonehoo 
I can give you the links to the NPM and changelogs for each package to put at the top of these highlights since they are already published - give me a few hours and i'll have those together for you to add.

**Comment by nicolethoen** [comment]
```
### Promoted package versions

- patternfly/patternfly ([changelog](https://github.com/patternfly/patternfly/releases/tag/v6.0.0))
  - [@patternfly/patternfly@6.0.0](https://www.npmjs.com/package/@patternfly/patternfly/v/6.0.0)


- patternfly/patternfly-react ([changelog](https://github.com/patternfly/patternfly-react/releases/tag/v6.0.0))
  - [@patternfly/react-charts@8.0.0](https://www.npmjs.com/package/@patternfly/react-charts/v/8.0.0)
  - [@patternfly/react-code-editor@6.0.0](https://www.npmjs.com/package/@patternfly/react-code-editor/v/6.0.0)
  - [@patternfly/react-core@6.0.0](https://www.npmjs.com/package/@patternfly/react-core/v/6.0.0)
  - [@patternfly/react-icons@6.0.0](https://www.npmjs.com/package/@patternfly/react-icons/v/6.0.0)
  - [@patternfly/react-styles@6.0.0](https://www.npmjs.com/package/@patternfly/react-styles/v/6.0.0)
  - [@patternfly/react-table@6.0.0](https://www.npmjs.com/package/@patternfly/react-table/v/6.0.0)
  - [@patternfly/react-tokens@6.0.0](https://www.npmjs.com/package/@patternfly/react-tokens/v/6.0.0)


- PatternFly extensions
  - [@patternfly/react-catalog-view-extension@6.0.0](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/6.0.0) ([changelog](https://github.com/patternfly/react-catalog-view/releases/tag/v6.0.0))
  - [@patternfly/react-console@6.0.0](https://www.npmjs.com/package/@patternfly/react-console/v/6.0.0) ([changelog](https://github.com/patternfly/react-console/releases/tag/v6.0.0))
  - [@patternfly/react-log-viewer@6.0.0](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/6.0.0) ([changelog](https://github.com/patternfly/react-log-viewer/releases/tag/v6.0.0))
  - [@patternfly/react-topology@6.0.0](https://www.npmjs.com/package/@patternfly/react-topology/v/6.0.0) ([changelog](https://github.com/patternfly/react-topology/releases/tag/v6.0.0))
  - [@patternfly/react-user-feedback@6.0.0](https://www.npmjs.com/package/@patternfly/react-user-feedback/v/6.0.0) ([changelog](https://github.com/patternfly/react-user-feedback/releases/tag/v6.0.0))
  - [@patternfly/react-virtualized-extension@6.0.0](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/6.0.0) ([changelog](https://github.com/patternfly/react-virtualized-extension/releases/tag/v6.0.0))
  - [@patternfly/quickstarts@6.0.0](https://www.npmjs.com/package/@patternfly/quickstarts/v/6.0.0) ([changelog](https://github.com/patternfly/patternfly-quickstarts/releases/tag/v6.0.0))
  - [@patternfly/virtual-assistant@2.0.0](https://www.npmjs.com/package/@patternfly/virtual-assistant/v/2.0.0) ([changelog](https://github.com/patternfly/patternfly-quickstarts/releases/tag/v2.0.0))
  - [@patternfly/react-component-groups@6.0.0](https://www.npmjs.com/package/@patternfly/react-component-groups/v/6.0.0) ([changelog](https://github.com/patternfly/react-component-groups/releases/tag/v6.0.0))

```

**Comment by nicolethoen** [comment]
- [@patternfly/react-topology@6.0.1](https://www.npmjs.com/package/@patternfly/react-topology/v/6.0.1) ([changelog](https://github.com/patternfly/react-topology/releases/tag/v6.0.1))

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 7]
```diff
@@ -7,6 +7,117 @@ section: get-started
...
+    - [@patternfly/react-styles@6.0.0](https://www.npmjs.com/package/@patternfly/react-styles/v/6.0.0)
+    - [@patternfly/react-table@6.0.0](https://www.npmjs.com/package/@patternfly/react-table/v/6.0.0)
+    - [@patternfly/react-tokens@6.0.0](https://www.npmjs.com/package/@patternfly/react-tokens/v/6.0.0)
+- PatternFly extensions
+    - [@patternfly/react-catalog-view-extension@6.0.0](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/6.0.0) ([changelog](https://github.com/patternfly/react-catalog-view/releases/tag/v6.0.0))
+    - [@patternfly/react-console@6.0.0](https://www.npmjs.com/package/@patternfly/react-console/v/6.0.0) ([changelog](https://github.com/patternfly/react-console/releases/tag/v6.0.0))
+    - [@patternfly/react-log-viewer@6.0.0](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/6.0.0) ([changelog](https://github.com/patternfly/react-log-viewer/releases/tag/v6.0.0))
+    - [@patternfly/react-topology@6.0.0](https://www.npmjs.com/package/@patternfly/react-topology/v/6.0.0) ([changelog](https://github.com/patternfly/react-topology/releases/tag/v6.0.0))
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:31]:
```suggestion
    -   - [@patternfly/react-topology@6.0.1](https://www.npmjs.com/package/@patternfly/react-topology/v/6.0.1) ([changelog](https://github.com/patternfly/react-topology/releases/tag/v6.0.1))

```

**Review by nicolethoen — APPROVED** [review]

**Review by dlabaj — APPROVED** [review]

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 8]
```diff
@@ -7,6 +7,118 @@ section: get-started
...
+- Data list
+    - Refactored `<DataListCheck>` to use `<Checkbox>` internally, which fixes broken checkboxes in the examples. 
+- Dropdown
+    - Added `focusTimeoutDelay`, which specifies the time to wait before setting focus on the first dropdown item when `shouldFocusFirstItemOnOpen` is set. 
+    - Fixed issues with invalid and jumpy scrolling when focusing on the first menu item.
+- Jump links
+    - Fixed improper offset in [demo](/components/jump-links/react-demos). 
+- Menu toggle
+    - Removed `pf-m-actions` and, consequently, `SplitButtonOptions`. Items should now be passed directly to `splitButtonItems`.
+    - Added `isPlaceholder` to support customizable placeholder text, as well as [a corresponding example](/components/menu-toggle#placeholder-text-in-toggle).
```
Comment (srambach) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:77]: Looks like this should be `/components/menus/menu-toggle/#placeholder-text-in-toggle`

File: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md [hunk 9]
```diff
@@ -7,6 +7,118 @@ section: get-started
...
+    - [@patternfly/react-virtualized-extension@6.0.0](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/6.0.0) ([changelog](https://github.com/patternfly/react-virtualized-extension/releases/tag/v6.0.0))
+    - [@patternfly/quickstarts@6.0.0](https://www.npmjs.com/package/@patternfly/quickstarts/v/6.0.0) ([changelog](https://github.com/patternfly/patternfly-quickstarts/releases/tag/v6.0.0))
+    - [@patternfly/virtual-assistant@2.0.0](https://www.npmjs.com/package/@patternfly/virtual-assistant/v/2.0.0) ([changelog](https://github.com/patternfly/patternfly-quickstarts/releases/tag/v2.0.0))
+    - [@patternfly/react-component-groups@6.0.0](https://www.npmjs.com/package/@patternfly/react-component-groups/v/6.0.0) ([changelog](https://github.com/patternfly/react-component-groups/releases/tag/v6.0.0))
+
+### PatternFly AI
+
+We're excited to introduce PatternFly AI: our new effort to support and integrate AI into our design system. We've added guidance and resources, contained within a new section of our website:
+
+- [About PatternFly AI](/patternfly-ai/about-patternfly-ai)
```
Comment (srambach) [inline packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md:43]: Looks like this URL is `/patternfly-ai/about-ai` now?

**Review by srambach — COMMENTED** [review]
Looking really good; just a couple of URLs that look like they need a slight adjustment.

**Review by srambach — APPROVED** [review]
Thanks for the fixes! LGTM 🎃

**Review by andrew-ronaldson — APPROVED** [review]
BAM! Awesome!

**Pull request merged**